docs/dask_cudf/source/api.rst

```diff
@@ -2,74 +2,35 @@
  API reference
 ===============
 
-This page provides a list of all publicly accessible modules, methods,
-and classes in the ``dask_cudf`` namespace.
+As recommended in the introduction, ``dask_cudf`` should be used through the
+`Dask DataFrame`_ API with the backend set to cudf.
 
+.. code-block:: python
+
+   >>> import dask
+   >>> dask.config.set({"dataframe.backend": "cudf"})
+
+With this configuration, ``dask_cudf`` will behave as documented in the Dask
```

Comment:
Something feels confusing about this sentence, but it may just be me.  `dask_cudf` implements the Dask DataFrame API whether or not the `"dataframe.backend"` configuration is set to `"cudf"`.

Maybe it would be best to start the sentence by clarifying this common point of confusion? E.g.

```
Dask cuDF implements the Dask-DataFrame API with with ``cudf`` objects used in place of
``pandas`` objects. As recommended in the introduction, the best way to use ``dask_cudf``
is to use the `Dask DataFrame`_ API with the backend set to cudf.

.. code-block:: python

   >>> import dask
   >>> dask.config.set({"dataframe.backend": "cudf"})
   
The rest of this page documents the API you might use from ``dask_cudf``
explicitly.
```